### PR TITLE
Add three tests PeoplePage and Refactor Mickael code

### PR DESCRIPTION
**Pull request by Wis13**
Add test to verify title page People. [Trello](https://trello.com/c/uxSUHQKC/32-peopletitle001)
Add test to verify all elements on the page People is visible. [Trello2](https://trello.com/c/JU2jX0dy/157-peopleallelementisvisible002)
Add test to verify all elements on the page People is clickable. [Trello3](https://trello.com/c/brsSPdpX/158-peopleallelementisclickable003)

Code refactoring .

### CODE REVIEW

File: tests/locators_dashboard_page.py [hunk 1]
```diff
@@ -56,12 +58,12 @@ class FooterLocators:
...
-    URL_NEW_VIEW = 'http://localhost:8080/newView'
-    URL_FOOTER_REST_API = 'http://localhost:8080/api/'
+    URL_NEW_ITEM = TestData.BASE_URL + 'view/all/newJob'
+    URL_PEOPLE = TestData.BASE_URL + 'asynchPeople/'
+    URL_BUILD_HISTORY = TestData.BASE_URL + 'view/all/builds'
+    URL_MANAGE_JENKINS = TestData.BASE_URL + 'manage'
+    URL_MY_VIEW = TestData.BASE_URL + 'me/my-views/view/all/'
+    URL_LOCKABLE_RESOURCES = TestData.BASE_URL + 'lockable-resources/'
+    URL_NEW_VIEW = TestData.BASE_URL + 'newView'
+    URL_FOOTER_REST_API = TestData.BASE_URL + 'api/'
```
Comment (Python2-QAForEveryone) [inline tests/locators_dashboard_page.py:68]: I agree with these changes

File: tests/test_people_page.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+from pages.PeoplePage import PeoplePage
+from tests.locators_people_page import PeoplePageLocator, URLLocators
+
+
+class TestPagePeople:
+    def test_people_title_001(self):
+        driver = PeoplePage(self.driver)
+        driver.go_to_page(URLLocators.URL_PEOPLE)
+        print(driver.get_title())
```
Comment (Python2-QAForEveryone) [inline tests/test_people_page.py:11]: no prints in repository code.please remove

File: tests/locators_people_page.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    locators_people_page = [PEOPLE, PEOPLE_LOGO, TABLE_USER_ID, TABLE_USER_ID_ARROW, TABLE_NAME, TABLE_NAME_ARROW,
+                            TABLE_LAST_COMMIT_ACTIVITY, TABLE_LAST_COMMIT_ACTIVITY_ARROW, TABLE_ON, TABLE_ON_ARROW,
+                            TABLE_USER_ID_USERS, TABLE_NAME_USERS, TABLE_ICON_LINK, TABLE_ICON_LINK_2]
+
+
+class LoginLocator:
+    USER = 'j_username'
+    PASSWD = 'j_password'
+    SUBMIT = 'Submit'
```
Comment (Python2-QAForEveryone) [inline tests/locators_people_page.py:36]: Could you please explain:
1. Where and how you are using those variables(USER,PASSWD,SUBMIT) 
2. Why you initiated them twice in the same module?

**Review by Python2-QAForEveryone — CHANGES_REQUESTED** [review]
> Я немного подредактировал код Михаила, убрал хардкодинг URL-страниц , ну сделал тесты всех элементов на странице is_clickable и is_visible. Да тесты на FireFox не проходят :(

Please include link to Trelo task

File: requirements.txt [hunk 4]
```diff
@@ -1,3 +1,4 @@
-pytest
-selenium
-webdriver_manager
\ No newline at end of file
+pytest~=6.2.4
+selenium~=3.141.0
+webdriver_manager
+requests~=2.25.1
```
Comment (Python2-QAForEveryone) [inline requirements.txt:4]: i agree with these changes, but next time do not include changes that not related to you TC or task.
in other words it should be done with separated PR

File: tests/locators_people_page.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
+from selenium.webdriver.common.by import By
+
+from config.TestData import TestData
+
+USER = 'j_username'
+PASSWD = 'j_password'
+SUBMIT = 'Submit'
+PEOPLE = (By.XPATH, '/html/body/div[4]/div[1]/div[1]/div[2]/span/a/span[2]')
+PEOPLE_LOGO = (By.XPATH, '/html/body/div[4]/div[2]/h1/img')
```
Comment (Python2-QAForEveryone) [inline tests/locators_people_page.py:9]: Maybe PEOPLE and PEOPLE_LOGO should be under class PeoplePageLocator:?

**Review by Python2-QAForEveryone — CHANGES_REQUESTED** [review]

File: tests/locators_dashboard_page.py (hunk 1 above)
Comment (Wis13) [reply tests/locators_dashboard_page.py:68]: Ок

File: requirements.txt (hunk 4 above)
Comment (Wis13) [reply requirements.txt:4]: Ok

File: tests/test_people_page.py (hunk 2 above)
Comment (Wis13) [reply tests/test_people_page.py:14]: Sorry, I don't understand what wrong?

File: tests/locators_people_page.py (hunk 3 above)
Comment (Wis13) [reply tests/locators_people_page.py]: I agree this is superfluous.

File: tests/locators_people_page.py (hunk 5 above)
Comment (Wis13) [reply tests/locators_people_page.py:7]: Of course!

File: tests/test_people_page.py (hunk 2 above)
Comment (maximashuev) [reply tests/test_people_page.py:14]: line 11
it's not a good practice to leave a print function in your code

File: config/TestData.py [hunk 6]
```diff
@@ -1,5 +1,5 @@
 class TestData:
 
-    BASE_URL = 'http://localhost:8080/'
+    BASE_URL = 'http://localhost:9090/'
```
Comment (maximashuev) [inline config/TestData.py:3]: why you changed the port number?

File: tests/locators_people_page.py [hunk 7]
```diff
@@ -30,11 +28,12 @@ class PeoplePageLocator:
...
-
 class URLLocators:
     URL_PEOPLE = TestData.BASE_URL + 'asynchPeople/'
+
+
+class IdUsersLocator:
+    TABLE_USER_ID_ARROW = (By.XPATH, '//*[@id="people"]/tbody/tr[1]/th[2]/a/span')
+    locator_id_user_arrow = [TABLE_USER_ID_ARROW]
+    TABLE_USER_ID = (By.XPATH, '//*[@id="people"]/tbody/tr[1]/th[2]/a')
+    locator_id_user = [TABLE_USER_ID]
```
Comment (maximashuev) [inline tests/locators_people_page.py:38]: what was the reason to create these variables "locator_id_user_arrow" and "locator_id_user"? 
i don't see where you using them.

**Review by maximashuev — CHANGES_REQUESTED** [review]

File: tests/test_people_page.py [hunk 8]
```diff
@@ -0,0 +1,27 @@
+import time
+
+import pytest
+
```
Comment (maximashuev) [inline tests/test_people_page.py:4]: remove line 1,2,4

**Review by maximashuev — CHANGES_REQUESTED** [review]

File: tests/test_people_page.py (hunk 2 above)
Comment (Wis13) [reply tests/test_people_page.py:14]: Ok

File: config/TestData.py (hunk 6 above)
Comment (Wis13) [reply config/TestData.py]: This is my mistake. I forgot to exclude TestData from the commit.

File: tests/locators_people_page.py (hunk 7 above)
Comment (Wis13) [reply tests/locators_people_page.py:39]: This is for future tests

File: tests/test_people_page.py (hunk 8 above)
Comment (Wis13) [reply tests/test_people_page.py:4]: Ok

**Review by maximashuev — APPROVED** [review]

**Review by Python2-QAForEveryone — APPROVED** [review]

**Pull request merged**